tests/network/utils.py

```diff
@@ -249,13 +249,13 @@ def assert_ssh_alive(ssh_vm, src_ip):
         sleep=1,
         func=run_ssh_commands,
         host=ssh_vm.ssh_exec,
-        commands=[shlex.split("sudo netstat -tulpan | grep ssh")],
+        commands=[shlex.split("sudo ss -tupan | grep ssh")],
```

Comment:
it can be used without grep:

```
ss -o state established '( dport = :22 )' --no-header
```

in this case you don't need check the output line by line